subsys/bluetooth/audio/bap_unicast_client.c

```diff
@@ -2089,7 +2089,8 @@ static void bt_audio_codec_qos_to_cig_param(struct bt_iso_cig_param *cig_param,
 {
 	cig_param->framing = qos->framing;
 	cig_param->packing = BT_ISO_PACKING_SEQUENTIAL; /*  TODO: Add to QoS struct */
-	cig_param->interval = qos->interval;
+	cig_param->c_to_p_interval = qos->interval;
+	cig_param->p_to_c_interval = qos->interval;
```

Comment:
Hmm, we may need to support individual qos->interval values as well for the BAP API, but that's a separate fix. Ditto with latency.

Reply:
This one I'd like to put in a separate PR